Lib/xml/etree/ElementTree.py

```diff
@@ -737,7 +737,7 @@ def write(self, file_or_filename,
                 if enc_lower == "unicode":
                     # Retrieve the default encoding for the xml declaration
                     import locale
-                    declared_encoding = locale.getpreferredencoding()
+                    declared_encoding = locale.getpreferredencoding(False)
```

Comment:
I keep using getpreferredencoding(False) here because this function should use UTF-8 in UTF-8 mode.

Reply:
Would you mind to write a separated PR for this change? It's not directly unrelated, and you should document it with a separated NEWS entry.